/usr/share/abuild/default.conf adds -fstack-clash-protection to CFLAGS,
but it is not recognized by Slimcc, so add it to the ignore list.

--- a/main.c
+++ b/main.c
@@ -512,6 +512,7 @@ static int parse_args(int argc, char **argv) {
         !strcmp(argv[i], "-fno-stack-protector") ||
         !strcmp(argv[i], "-fno-strict-aliasing") ||
         !strcmp(argv[i], "-fno-strict-overflow") ||
+        !strcmp(argv[i], "-fstack-clash-protection") ||
         !strcmp(argv[i], "-fwrapv") ||
         !strcmp(argv[i], "-m64") ||
         !strcmp(argv[i], "-mfpmath=sse") ||
